### DISCUSSION TITLE
Dataset always rewrite

### DISCUSSION THREAD

**Discussion by din9w** [pyg-team/pytorch_geometric#4436] [Q&A] [answered]
I have already created my own datasets und these were processed. But when I loaded the dataset like this 
dataset = SensorDataset(root= 'D:\projekt\wd \data')
it will process again. So how can I without rewriting load datasets?

**Comment by rusty1s** [comment, marked as answer]
You should specify via
```python
@property
def processed_file_names(self) -> List[str]:
    return [...]
```
which file names should be found on disk inside `{root_dir}/processed` in order to skip processing. Let me know if this works!

**Reply by din9w** [reply]
It works, thx!
